docs/working_docs/output.md

```diff
@@ -26,12 +31,52 @@ Sample output:
 │ https://osv.dev/GHSA-m5pq-gvj9-9vr8 │ crates.io │ regex                    │ 1.3.1   │ path/to/Cargo.lock │
 ╰─────────────────────────────────────┴───────────┴──────────────────────────┴─────────┴────────────────────╯
 ```
+</details>
+
+---
+
+### Markdown Table
+
+```bash
+osv-scanner --format markdown your/project/dir
+```
+
+<details markdown="1">
+<summary><b>Sample markdown output</b></summary>
+
+**Raw output:**
+
+```
+| OSV URL | Ecosystem | Package | Version | Source |
+| --- | --- | --- | --- | --- |
+| https://osv.dev/GHSA-c3h9-896r-86jm<br/>https://osv.dev/GO-2021-0053 | Go | github.com/gogo/protobuf | 1.3.1 | ../scorecard-check-osv-e2e/go.mod |
+| https://osv.dev/GHSA-m5pq-gvj9-9vr8<br/>https://osv.dev/RUSTSEC-2022-0013 | crates.io | regex | 1.5.1 | ../scorecard-check-osv-e2e/sub-rust-project/Cargo.lock |
+```
+
+**Rendered:**
 
-### `json` format
+| OSV URL | Ecosystem | Package | Version | Source |
+| --- | --- | --- | --- | --- |
+| https://osv.dev/GHSA-c3h9-896r-86jm<br/>https://osv.dev/GO-2021-0053 | Go | github.com/gogo/protobuf | 1.3.1 | ../scorecard-check-osv-e2e/go.mod |
+| https://osv.dev/GHSA-m5pq-gvj9-9vr8<br/>https://osv.dev/RUSTSEC-2022-0013 | crates.io | regex | 1.5.1 | ../scorecard-check-osv-e2e/sub-rust-project/Cargo.lock |
 
-Outputs the results as a JSON object to stdout, with all other output being directed to stderr - this makes it safe to redirect the output to a file with `osv-scanner --format json ... > /path/to/file.json`.
+</details>
+
+---
 
-Sample output:
+### JSON
+
+```bash
+osv-scanner --format json your/project/dir
+```
+
+Outputs the results as a JSON object to stdout, with all other output being directed to stderr - this makes it safe to redirect the output to a file with
+```bash
+osv-scanner --format json ... > /path/to/file.json
```

Comment:
I don't like the three dots here, it isn't clear enough. 

I assume the full command is:

osv-scanner --format json your/project/dir >/path/to/file.json

But it feels like I am guessing.

Reply:
Good point, updated to have a full command